README.md

```diff
@@ -1,4 +1,43 @@
 # JanusGraph Ambari plugin
+JanusGraph is a scalable graph database optimized for storing and querying graphs containing hundreds of billions of vertices and edges distributed across a multi-machine cluster. JanusGraph is a transactional database that can support thousands of concurrent users executing complex graph traversals in real time. 
+
+This plugin has been tested with Ambari versions 2.5, and 2.6. It is not compatible with Ambari version 2.7.
+
+Instructions on how to add JanusGraph to [Apache Ambari](https://ambari.apache.org ) or [Hortonworks Data Platform(HDP)](https://hortonworks.com/products/data-platforms/hdp/) as a service can be found below.
+
+
+- To download the service and link it to Ambari run the following commands
+```
+VERSION=`hdp-select versions | head -n1 | cut -d'.' -f1,2`
+git clone https://github.com/JanusGraph/janusgraph-ambari.git /var/lib/ambari-server/resources/stacks/HDP/$VERSION/services/JANUSGRAPH
+```
+
+```
+sudo service ambari-server restart
+```
+## Service Installation
+- Once the ambari-server service has restarted login into the webui and click on the Actions dropdown on the lower lefthand side and select '+Add Service'.
+Click on the checkbox next to Janusgraph and then press next. 
+Currently JanusGraph needs to be located on the same host as Solr. If solr is not yet installed you'll need to install it through the [mpack](https://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.6.4/bk_solr-search-installation/content/ch_hdp-search-install-ambari.html).
```

Comment:
So, JanusGraph Ambari needs Solr as the index backend? I don't know Ambari at all, so this might be a stupid question: Is that obvious to potential users of this plugin? Otherwise I think that a short description in the beginning of this document would be helpful that explains the architecture supported by this plugin.